src/screencast/wlr_screencast.c

```diff
@@ -190,7 +190,7 @@ static void wlr_frame_ready(void *data, struct zwlr_screencopy_frame_v1 *frame,
 
 	if (!cast->quit && !cast->err && cast->pwr_stream_state) {
 		pw_loop_signal_event(cast->ctx->state->pw_loop, cast->event);
-		return ;
+		return;
```

Comment:
This change is completly unrelated. Can you remove it from the patch.

Reply:
Split into its own commit.